code/modules/jobs/job_types/internal_affairs_agent.dm

```diff
@@ -0,0 +1,62 @@
+/datum/job/iaa
+	title = "Internal Affairs Agent"
+	flag = IAA
+	department_head = list("Captain")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "Central Command"
+	selection_color = "#ddddff"
+	chat_color = "#50C878"
+	exp_requirements = 840
+	exp_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/iaa
+
+	access = list(ACCESS_IAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO)
+	minimal_access = list(ACCESS_IAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO)
+	paycheck = PAYCHECK_MEDIUM
+	paycheck_department = ACCOUNT_CIV
+	mind_traits = list(TRAIT_LAW_ENFORCEMENT_METABOLISM)
+
+	display_order = JOB_DISPLAY_ORDER_IAA
+	departments = DEPARTMENT_COMMAND | DEPARTMENT_SERVICE
+	rpg_title = "Diplomat"
+
+	species_outfits = list(
+		SPECIES_PLASMAMAN = /datum/outfit/plasmaman/lawyer
+	)
+
+/datum/outfit/job/iaa
+	name = "Internal Affairs Agent"
+	jobtype = /datum/job/iaa
+
+	id = /obj/item/card/id/job/iaa
+	belt = /obj/item/pda/iaa
+	ears = /obj/item/radio/headset/headset_iaa
+	uniform = /obj/item/clothing/under/suit/black
+	suit = /obj/item/clothing/suit/toggle/lawyer/black
+	shoes = /obj/item/clothing/shoes/laceup
+	l_hand = /obj/item/storage/briefcase/lawyer
+	l_pocket = /obj/item/laser_pointer
+	r_pocket = /obj/item/assembly/flash/handheld
```

Comment:
🙄

Reply:
The original intent was to provide bare bones self defense considering the fact that the CAA would often be in an area of a department they don't have access to, but after talking with some admins about it although it would make sense for them to have it (and it would fit the theme) it would probably lead to accidental validhunting. I've decided it would be best to remove it from them.